src/System.Net.NetworkInformation/tests/FunctionalTests/NetworkInterfaceBasicTest.cs

```diff
@@ -86,8 +86,8 @@ public void BasicTest_AccessInstanceProperties_NoExceptions_Linux()
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.OSX)]  // Some APIs are not supported on OSX
-        public void BasicTest_AccessInstanceProperties_NoExceptions_Osx()
+        [PlatformSpecific(TestPlatforms.OSX|TestPlatforms.FreeBSD)]  // Some APIs are not supported on OSX
```

Comment:
Some of these comments reference OSX.  I'd just remove the comments rather than update them.